src/editable-heading/editable-heading.tsx

```diff
@@ -34,6 +36,8 @@ export type EditableHeadingProps = Omit<InputHTMLAttributes<HTMLInputElement>, '
   'data-test'?: string | null;
   error?: string;
   multiline?: boolean;
+  multilineInput?: boolean;
```

Comment:
What would you think of using single `multiline` prop for that purpose? Otherwise it may be weird to have `multiline={false} multilineInput={true}`, isn't it?

Reply:
Yes I agree with you, I wanted to keep backwards compatibility so that if someone was already using multiline separately, it wouldn't be a surprise to them that it's now also multiline editing. But in general, I can use it as a single flag

Reply:
I understand the reasoning, thank you.

But in this specific case we may say "having no multilines in Input mode is a kind of bug", because I believe it is the expected behavior to have it. So I'd vote to have a single prop

Reply:
I think this is the last point left to address

Reply:
done